researcher.lic

```diff
@@ -0,0 +1,183 @@
+=begin
+  Documentation: https://elanthipedia.play.net/Lich_script_repository#check-symbiosis
+=end
+begin
+  custom_require.call %[common common-arcana drinfomon events]
+rescue
+  echo "Dependency is not running! This script requires dependency to be installed and running."
```

Comment:
Now that this script will be part of our Lich framework, lines 4-9 can be replaced by just the custom_require line; the rest aren't necessary

Reply:
wasn't sure there, so I just left it, will make the change